integration/util/scenario/ctx.js

```diff
@@ -34,10 +37,24 @@ class Ctx {
     return process.env['INITIAL_YIELD'] || this.scenInfo['initial_yield'] || 0;
   }
 
-  __initialYieldStart() {
+  __initialYieldStartRaw() {
     return process.env['INITIAL_YIELD_START'] || this.scenInfo['initial_yield_start'] || this.startTime;
   }
 
+  __initialYieldStartMS() {
+    let raw = this.__initialYieldStartRaw();
+    if (raw > 4102444800) { // Jan 1, 2100 as seconds since 1970
```

Comment:
Basically, we're just trying to see if the time is so large that it _must_ have been ms and not s